spec/spec_helper.rb

```diff
@@ -1,6 +1,17 @@
+# Set the environment to "test"
+ENV['RACK_ENV'] = 'test'
```

Comment:
I wonder whether it would be a good idea to anticipate having two databases at this point that link up to the program, one for testing.